themes/app/src/scss/_typography.scss

```diff
@@ -72,62 +72,72 @@ $font-path: './webfonts';
         url('#{$font-path}/SourceSansPro-BoldItalic.woff') format('woff');
 }
 
-h1 {
+body {
     font-family: 'SourceSansPro', sans-serif;
+}
+
+@for $index from 1 through 6 {
+    h#{$index} {
+        margin: 8px 0;
+    }
+}
+
+@for $index from 2 through 6 {
+    h#{$index} {
+        color: $grey;
+    }
+}
+
+h1 {
     font-weight: 700;
     font-size: 32px;
```

Comment:
one thing I missed was turning these `font-sizes` into variables, same with `line-heights` it will make it a lot easier to customise later on as you will only have to change the value in one place so you would have something like `font-size-s`, `font-size-m`, `line-height-s` etc...

Once you've added these, apply them accross your styles :)